nrpy/infrastructures/CarpetX/general_relativity/Ricci_eval.py

```diff
@@ -0,0 +1,128 @@
+"""
+Generates function to compute the Ricci tensor for use in the BSSN evolution equations.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+
+import nrpy.c_codegen as ccg
+import nrpy.c_function as cfc
+import nrpy.grid as gri
+import nrpy.params as par
+import nrpy.helpers.parallel_codegen as pcg
+import nrpy.finite_difference as fin
+
+import nrpy.infrastructures.CarpetX.simple_loop as lp
+from nrpy.equations.general_relativity.BSSN_quantities import BSSN_quantities
+
+standard_ET_includes = [
```

Comment:
Centralize.

Reply:
Centralized in `CarpetX_include_header.py`